README.md

```diff
@@ -25,22 +25,27 @@ wasm-bindgen = { version = "0.2" }
 ```rust
 use serde::{Deserialize, Serialize};
 use tsify::Tsify;
+use tsify::Ts;
 use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsError;
 
 #[derive(Tsify, Serialize, Deserialize)]
-#[tsify(into_wasm_abi, from_wasm_abi)]
 pub struct Point {
     x: i32,
     y: i32,
 }
 
 #[wasm_bindgen]
-pub fn into_js() -> Point {
-    Point { x: 0, y: 0 }
+pub fn into_js() -> Result<Ts<Point>, JsError> {
+    let point = Point { x: 0, y: 0 };
+    Ok(point.into_ts()?)
 }
 
 #[wasm_bindgen]
-pub fn from_js(point: Point) {}
+pub fn from_js(point: Ts<Point>) -> Result<(), JsError> {
+    let point: Point = point.to_rust()?;
```

Comment:
I initially called this `.into_rust()` but changed to `.to_rust()` when I made it take &self. But the other APIs (Tsify trait) do not follow this convention, so maybe this is more confusing and it would be better off called `into_rust`.